css/main.css

```diff
@@ -363,20 +395,18 @@ and we don't want the map to expand down past screen.
 I feel like there should be a way to do this but I can't figure it out right now
 */
 .mapContainer{
-	position: fixed;
 	display: flex;
 	flex-direction: column;
-	padding:1em;
 	width: 100%;
 	height: 100%;
-	top: 0.5em;
+	top: 0;
 	left: 0;
```

Comment:
we don't need top or left, since we removed the fixed position.

Reply:
There are tons of these in here, I think I may have just not understood how they worked when I first started out and left them there from how they were set up before. I removed all instances of unnecessary top and left values from all css files.